src/invision/components/add-movie/add-movie.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import Modal from 'react-awesome-modal';
+
+export default class MoviePopup extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      title: this.props.movie?.title,
+      releaseDate: this.props.movie?.releaseDate,
+      url: this.props.movie?.url,
+      genre: this.props.movie?.genre,
+      overview: this.props.movie?.overview,
+      runtime: this.props.movie?.runtime
+    }
+    this.handleTitleChange = this.handleTitleChange.bind(this);
+    this.handleReleaseDateChange = this.handleReleaseDateChange.bind(this);
+    this.handleUrlChange = this.handleUrlChange.bind(this);
+    this.handleGenreChange = this.handleGenreChange.bind(this);
+    this.handleOverviewChange = this.handleOverviewChange.bind(this);
+    this.handleRuntimeChange = this.handleRuntimeChange.bind(this);
+  }
+
+  handleTitleChange(event) {
+    this.setState({ title: event.target.value });
+  }
+
+  handleReleaseDateChange(event) {
+    this.setState({ releaseDate: event.target.value });
+  }
+
+  handleUrlChange(event) {
+    this.setState({ url: event.target.value });
+  }
+  handleGenreChange(event) {
+    this.setState({ genre: event.target.value });
+  }
+
+  handleRuntimeChange(event) {
+    this.setState({ runtime: event.target.value });
+  }
+
+  handleOverviewChange(event) {
+    this.setState({ overview: event.target.value });
+  }
+
+  closeModal(status) {
+    this.visible = status;
+  }
+
+  render() {
+    return (
+      <Modal visible={this.props.visible} width="600" height="auto" effect="fadeInUp" onClickAway={() => this.closeModal()}>
```

Comment:
visible property change from the parent doesn't reflect here. 
I think that is why popup is not visible.